mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -53,7 +53,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
 
-    private int batchSize = 64;
+    private final int batchSize = 64;
```

Comment:
Lets leave it in there for now. There is a ticket to move that to
configuration.

On Wed, May 25, 2016 at 7:41 AM, Brandon Vulaj <notifications@github.com>
wrote:

> In mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/275#discussion_r64573881>
> :
>
> > @@ -53,7 +53,7 @@
> >
> >      private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
> >
> > -    private int batchSize = 64;
> > +    private final int batchSize = 64;
>
> It's definitely a configuration candidate, but @bserdar
> <https://github.com/bserdar> and I just decided to leave it here for now
> and move it out in another PR. Is there a place in LB today to wire up
> configuration properties?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/275/files/ed3af7895cc087df19ff1c158c8fa74144ceb734#r64573881>
>